src/test/java/uk/gov/companieshouse/api/accounts/transformer/CurrentPeriodTransformerTest.java

```diff
@@ -49,12 +53,21 @@ public void testRestToEntityTransformerWithPopulatedObject() {
 
         BalanceSheet balanceSheet = new BalanceSheet();
         balanceSheet.setCalledUpShareCapitalNotPaid(CALLED_UP_SHARE_CAPITAL_NOT_PAID_VALID);
+
         FixedAssets fixedAssets = new FixedAssets();
         fixedAssets.setTangible(TANGIBLE_VALID);
         fixedAssets.setTotalFixedAssets(FIXED_ASSETS_TOTAL_VALID);
 
         balanceSheet.setFixedAssets(fixedAssets);
 
+        CurrentAssets currentAssets = new CurrentAssets();
+        currentAssets.setStocks(CURRENT_ASSETS_VALID);
+        currentAssets.setDebtors(CURRENT_ASSETS_VALID);
+        currentAssets.setCashAtBankAndInHand(CURRENT_ASSETS_VALID);
+        currentAssets.setTotalCurrentAssets(CURRENT_ASSETS_TOTAL_VALID);
+
+        balanceSheet.setCurrentAssets(currentAssets);
```

Comment:
This method is going to keep growing every single time we add a new note.  We could move the creations to a separate methods. 

```
balanceSheet.setCurrentAssets(createCurrentAssets());
```

```
CurrentAssets createCurrentAssets() {
        CurrentAssets currentAssets = new CurrentAssets();
        currentAssets.setStocks(CURRENT_ASSETS_VALID);
        currentAssets.setDebtors(CURRENT_ASSETS_VALID);
        currentAssets.setCashAtBankAndInHand(CURRENT_ASSETS_VALID);
        currentAssets.setTotalCurrentAssets(CURRENT_ASSETS_TOTAL_VALID);

return currentAssets;
}
```

We could do the same for the existing code. What do you think?